### ISSUE TITLE
Modularise SCSS file. 

### ISSUE THREAD

**Issue by Panoply** [Shopify/Timber#503]
If anyone is looking for a modularized version of the `timber.scss.lqiuid` file I've put something together using a couple of task runners and bower:

`[https://github.com/Panoply/timber-modular](https://github.com/Panoply/timber-modular)`

I noticed a couple of issues raised about this, particularly (#155 ) however they were closed and put in the pipeline?  On a couple of recent projects I only required a few components from timber and wanted to keep it under version control, i'd overwrite the variables using `settings_data.json` and any config settings I'd include in a separate file, so there is some optimisation required to it if you want Shopify to compress the scss and include certain tags etc. 

The end result is a sass directory structure similar to bootstrap with only a couple of issues, which i've listed in the [readme.md](https://github.com/Panoply/timber-modular/blob/master/readme.md).

Though it might be helpful to someone!

**Comment by suture** [comment]
It's a good idea. I can see the same problem with Timber JS. 

We end up load stuff that we don't need, or manually hacking it out.

**Comment by Panoply** [comment]
Timber JS is pretty lean, only contains small logics, so you can get away with that one imo, but you could throw it in your compression stream.  Here is the Timber 2.1.5 split into multiple: https://github.com/Panoply/timber-modular/tree/master/src

**Comment by cameronroe** [comment]
This is a shame, without modular partials, the code is messy and difficult to iterate on effectively. Any chance this will be supported in the near future?

**Comment by Cam** [comment]
@cameronroe I don't think it's hard to manage at all. The only issue I have with the long SCSS file method is the long upload time for it via the API. But there are ways around that too.

**Comment by cameronroe** [comment]
@Cam Anything monolithic is hard to reason about. Sad to hear that you'd rather keep everything in one file.. doesn't seem modular at all. :/

**Comment by Cam** [comment]
@cameronroe I'm not saying it's perfect. I just don't think it is hard to manage. If you structure your file well, you shouldn't have any issues. You can create a modular setup of your own locally, then just let it compile on-the-fly for uploading to Shopify.

**Comment by cameronroe** [comment]
Yeah, I've already created a modular set up for me, but I think it could definitely improve Timber by having partials. You can use cmd+f to search for that section, but it still makes sense to use partials because it helps people see the actual files all in the same place. In the code you have to search around and it's less verbose.
